waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/TunnelableFactorySupplierTest.java

```diff
@@ -110,4 +110,10 @@ public void incorrectStrictHostKeyCheckingSetting() {
     supplier.get(hiveConf);
   }
 
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
I don't understand why this would throw the exception? "no" is valid input is it not? Or have you not got up to here yet?

Reply:
That's true, it doesn't throw an exception! I changed it :)